pkg/sql/parser/parse_test.go

```diff
@@ -2595,6 +2598,9 @@ SKIP_MISSING_FOREIGN_KEYS, SKIP_MISSING_SEQUENCES, SKIP_MISSING_SEQUENCE_OWNERS,
 
 		{`ALTER TYPE t OWNER TO CURRENT_USER`, "ALTER TYPE t OWNER TO \"current_user\""},
 		{`ALTER TYPE t OWNER TO SESSION_USER`, "ALTER TYPE t OWNER TO \"session_user\""},
+
+		{`REASSIGN OWNED BY CURRENT_USER TO foo`, "REASSIGN OWNED BY \"current_user\" TO foo"},
+		{`REASSIGN OWNED BY SESSION_USER TO foo`, "REASSIGN OWNED BY \"session_user\" TO foo"},
```

Comment:
you can use backticks to avoid quotes, e.g.

```
`REASSIGNED OWNED BY "session_user" TO foo`
```

Reply:
Done in most recent commit (and edited the quotes in the ALTER TYPE lines above as well) 👍